hack/runtime-migrator/internal/runtime/migrator.go

```diff
@@ -145,10 +145,26 @@ func getAdministratorsList(ctx context.Context, provider kubeconfig.Provider, sh
 	})
 
 	subjects := make([]string, 0)
-
 	for _, clusterRoleBinding := range clusterRoleBindings.Items {
-		for _, subject := range clusterRoleBinding.Subjects {
-			subjects = append(subjects, subject.Name)
+		// We are interested only in cluster-admin role
```

Comment:
I think it makes sense to add the labeling logic to some separate function. Now it is a part of `getAdministratorsList` - so this functions gets and modifies the CRBs.